app/models/user.rb

```diff
@@ -0,0 +1,21 @@
+class User < ActiveRecord::Base
+    
+  def self.find_or_create_from_auth(auth)
```

Comment:
Assignment Branch Condition size for find_or_create_from_auth is too high. [40.01/15]